packages/spright-components/src/chat/conversation/styles.ts

```diff
@@ -18,8 +22,19 @@ export const styles = css`
         flex-direction: column;
         justify-content: flex-start;
         row-gap: 32px;
-        padding: ${mediumPadding};
-        background: ${applicationBackgroundColor};
+        padding: ${mediumPadding} ${standardPadding} ${mediumPadding}
+            ${standardPadding};
+        border: ${borderWidth} solid ${applicationBackgroundColor};
```

Comment:
playing in devtools it seems odd to me that the border is just around the the top half of the control (messages) instead of the full control. Should this be on `:host` instead?